cypress.config.js

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
```

Comment:
It looks like the the usage of ESM is causing the tests to error in CI, shall we just use commonjs?

```suggestion
const { defineConfig } = require('cypress');

module.exports = defineConfig({
```